pkg/storage/fs/ocis/upload.go

```diff
@@ -32,120 +32,55 @@ import (
 	"github.com/cs3org/reva/pkg/appctx"
 	"github.com/cs3org/reva/pkg/errtypes"
 	"github.com/cs3org/reva/pkg/logger"
+	"github.com/cs3org/reva/pkg/storage/utils/chunking"
 	"github.com/cs3org/reva/pkg/user"
 	"github.com/google/uuid"
 	"github.com/pkg/errors"
-	"github.com/rs/zerolog/log"
 	tusd "github.com/tus/tusd/pkg/handler"
 )
 
 var defaultFilePerm = os.FileMode(0664)
 
-// TODO deprecated ... use tus
-
 func (fs *ocisfs) Upload(ctx context.Context, ref *provider.Reference, r io.ReadCloser) (err error) {
-
-	var n *Node
-	if n, err = fs.lu.NodeFromResource(ctx, ref); err != nil {
-		return
-	}
-
-	// check permissions
```

Comment:
you removed the permissions checks ...

Reply:
I refactored the Upload method to work with the logic implemented in InitiateFileUpload, so that is taken care of there itself. I'm pushing a fix so that it works without calling that first as well.